randovania/data/json_data/prime1.txt

```diff
@@ -3070,36 +3070,26 @@ Asset id: 4272124642
   * Ice Door to Dynamo Access/Dock to Central Dynamo
   > Dock to Save Station Mines B
       After Security Drone Defeated
-  > Event - Security Drone
-      Energy Tank or Combat (Beginner)
+  > Drone Fight Trigger
+      Trivial
 
 > Dock to Quarantine Access A; Heals? False
   * Ice Door to Quarantine Access A/Dock to Central Dynamo
   > Dock to Save Station Mines B
-      Power Bomb and Morph Ball and After Security Drone Defeated
-  > Event - Security Drone
-      All of the following:
-          Power Bomb and Space Jump Boots and Morph Ball
-          Any of the following:
-              X-Ray Visor
-              Combat (Beginner) and Invisible Objects (Beginner)
+      Power Bomb and Morph Ball
 
 > Dock to Save Station Mines B; Heals? False
   * Ice Door to Save Station Mines B/Dock to Central Dynamo
   > Dock to Dynamo Access
-      All of the following:
-          After Security Drone Defeated
-          Any of the following:
-              Space Jump Boots
-              Morph Ball Bomb and Morph Ball and Bomb Jump (Expert)
+      Trivial
```

Comment:
Since the Save Station door connects to the Drone Trigger node and that node connects to Dynamo Access, the connection to Dynamo Access from this node should be removed. Outside of that simplification, listing it as trivial is wrong anyway.

Reply:
Good catch. I've removed the connection